test/sync/src/androidTest/kotlin/io/realm/test/mongodb/shared/SyncSessionTests.kt

```diff
@@ -65,20 +63,18 @@ class SyncSessionTests {
         user = runBlocking {
             app.createUserAndLogIn(email, password)
         }
-        tmpDir = PlatformUtils.createTempDir()
```

Comment:
Are you sure it is safe to just not use the temporary dir? How is cleanup done then? I would have assumed that the `TestApp` then needs to define this? Or we have some other logic that deletes the default folder after each test?

Reply:
Yeah, I noticed that too. I sort of left it in limbo while I was working on implementing the rest. I will sort it out.

Reply:
Turns out cleanup is done correctly when we call `TestApp.close()`. Also and just for fun I changed the deletion logic so that we use `PlatformUtils.deleteTempDir` instead of OKIO's `FileSystem.deleteRecursively` and it also works fine. If I'm honest, I don't recall why we reached the conclusion that we needed to use OKIO 😑  although after this PR it might make sense to use it if we keep having the need for handling more IO scenarios.